src/videos.md

```diff
@@ -69,6 +88,86 @@ Alvaro Sánchez Villar of the Princeton Plasma Physics Laboratory presented "MFE
+##### **October 25, 2022** | [MFEM Workshop 2022](workshop.md)
+
+![YouTube](tzQUlm0hlvE)
+
+Tim Brewer, Ben Shields, Peter Vonk, Jeff Heylmun, and Barlev Raymond of Synthetik Applied Technologies presented "blastFEM: A GPU-Accelerated, Very High-Performance and Energy-Efficient Solver for Highly Compressible Flows." Highly compressible multiphase and reactive flows are important, and manifest across a myriad of practical applications: novel energy production and propulsion methods, building design, safety and energy efficiency, material discovery, and maintenance of our nuclear arsenal. There are, however, few tools available to industry capable of simulating these flows at a resolution and scale suitable make predictions of adequate detail—at least within reasonable timeframes and budgetary constraints—to inform engineers and designers. A next generation, highly efficient simulation code is needed that can deliver results within useful timeframes, with sufficient detail to be useful to support simulation-driven design, discovery, and optimization. Furthermore, the code must be designed to run on modern and emerging heterogeneous architectures, and can efficiently leverage these architectures though the use of numerical schemes designed to maximized computational efficiency.
+
+---
+
+#### Adolfo Rodriguez (OpenSim Technology)
+#### *Using MFEM for Wellbore Stability Analysis*
+##### **October 25, 2022** | [MFEM Workshop 2022](workshop.md)
+
+![YouTube](gBEApU1V_80)
+
+Adolfo Rodriguez of OpenSim Technology presented "Using MFEM for Wellbore Stability Analysis." He discussed the results from a Department of Energy Small Business Innovation Research project regarding the implementation of wellbore stability analysis for hydrocarbon producing wells.
+
+---
+
+#### Julian Andrej (LLNL)
+#### *AWS Tutorial*
+##### **October 25, 2022** | [MFEM Workshop 2022](workshop.md)
+
+![YouTube](tsr6vza_7DM)
+
+In this tutorial, Julian Andrej of LLNL demonstrated how to use MFEM in the cloud (e.g., an Amazon Web Services instance) for scalable finite element discretization application development. Step-by-step instructions for the tutorial can be found at https://mfem.org/tutorial/.
+
+---
+
+#### Aaron Fisher (LLNL)
+#### *Wrap-Up and Simulation Contest Winners*
+##### **October 25, 2022** | [MFEM Workshop 2022](workshop.md)
+
+![YouTube](7uExaQm8Dmk)
+
+Aaron Fisher of LLNL concluded the workshop by announcing the winners of the simulation and visualization contest: (1) streamlines of the electric field generated by a current dipole source located in the temporal lobe of an epilepsy patient, rendered by Ben Zwick of the University of Western Australia; (2) a topology-optimized heat sink, rendered by Tobias Duswald of CERN/Technical University of Munich; (3) the magnetic field induced by current running through copper wire in air, rendered by Will Pazner of Portland State University. Contest winners are featured at https://mfem.org/gallery/.
```

Comment:
`https://mfem.org/gallery/` doesn't show up as a link for me. Maybe replace by

- `[https://mfem.org/gallery/](https://mfem.org/gallery/)` 

or

- `in the MFEM [Gallery](https://mfem.org/gallery/)`